.github/ISSUE_TEMPLATE/introduce-sgid-dataset.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Introduce data to the SGID
+about: Creation Checklist
+title: Add <dataset name> from <place>
+labels: 'introduction'
+assignees: ''
+
+---
+# Summary
+
+A short summary of the situation...
+
+<!-- 
+1. The three teams will triage the request, striking through any line that does not apply.
+1. Assign each step to someone based on list of assignments
+1. All checkboxes should either be striked out or checked before closing the issue.
+-->
+
+# Triage
+
+- [ ] Data Team Triage
+- [ ] Dev Team Triage (assigned to)
+- [ ] Cadastre Team Triage
+
+# Data is live in the following areas
+
+- [ ] ArcGIS Online (assigned to)
+- [ ] Internal SGID (assigned to)
+- [ ] SGID10 (assigned to)
+- [ ] Open SGID (assigned to)
+- [ ] Open data (assigned to)
+- [ ] gis.utah.gov data pages (assigned to)
+- [ ] Stewardship doc and sgid-index (assigned to)
+- [ ] `SGID.META.AGOLItems` (assigned to)
+- [ ] `AGRC_Shelved/AGOLItems_Shelved` (in AGOL) (assigned to)
+- [ ] `SGID.META.ChangeDetection` (assigned to)
```

Comment:
I'm not sure what this section means- Data should be put here? Data already exist here?

Reply:
Or by "draft to hack on" you mean we should all be pulling and making edits?

Reply:
I thought it meant that, yes, data should be placed there (as long as the one performing triage does not strike through) and the assignee checks it once the data is/are "live" in that spot.

Reply:
> Or by "draft to hack on" you mean we should all be pulling and making edits?

Yes, please add suggestions/review comments/etc to create and improve this process

Reply:
New tables are automatically added to the change detection table, no?
https://github.com/agrc/cambiador/blob/master/Program.cs#L22

Reply:
They should be now. 

We had the issue a couple of days ago where this was not working properly. This checklist would describe what was done, where things should end up, and that they did end up where they should.